app/views/activities/new.html.slim

```diff
@@ -0,0 +1,14 @@
+
+= form_for @activity do |f|
+  fieldset
+    - if @activity.errors.any?
```

Comment:
@jschmid1  that could be the idea http://asciicasts.com/episodes/211-validations-in-rails-3